Component Plug-in (CP) Examples/mapField/mapField/v1/index.html

```diff
@@ -39,22 +42,34 @@
     });
 
     function initMap() {
+      Appian.Component.setValidations('');
```

Comment:
Do you need this initial validations clear? I assume it's for the case where you update the API key, but clear the location - just want to confirm

Reply:
Yes this is to clean up the validation in the case that they've added an API key but no location is present yet :)